src/app/pages/quiz-page/test-info/test-info.component.html

```diff
@@ -4,33 +4,51 @@
   </mat-card-header>
   <mat-card-content>
     <p *ngIf="testing?.intervieweeName">
-      Interviewee: {{ testing?.intervieweeName }}
+      {{ "pages.quiz-page.test-info.interviewee" | translate }}:
```

Comment:
You can pass a parameter to your translation.
Pattern how to do this:
`"your_translation_key": "You translation message {parametherName}"`

And then use this translation as:
```
{{
            'translation_file.your_translation_key'
              | translate: { parametherName: variable.property }
          }}

```

Reply:
Please go through the project and change all places where you have to split a single message into two or more.

Reply:
Changed to strings with parameters